lib/agent.js

```diff
@@ -64,7 +64,9 @@ class Agent {
                 agent.connected = true
                 console.log(`connected to ${agent.creds.host}:${agent.creds.port} as ${options.clientId}`)
                 agent.error = null
-                agent.client.subscribe('#')
+                const topic = agent.creds.topicPrefix || '#'
+                console.log(`subscribing to ${topic}`)
```

Comment:
```suggestion
                console.log(`subscribing to "${topic}"`)
```

As this logging is deliberate, do you mind if we enclose it in single or double quotes so that we can see when `# ` or ` #` has been mistakenly entered?  Appended spaces often occur when copy/pasting strings.